htdocs/install/mysql/data/llx_c_action_trigger.sql

```diff
@@ -109,6 +109,7 @@ insert into llx_c_action_trigger (code,label,description,elementtype,rang) value
 insert into llx_c_action_trigger (code,label,description,elementtype,rang) values ('MEMBER_EXCLUDE','Member excluded','Executed when a member is excluded','member',27);
 insert into llx_c_action_trigger (code,label,description,elementtype,rang) values ('FICHINTER_VALIDATE','Intervention validated','Executed when a intervention is validated','ficheinter',30);
 insert into llx_c_action_trigger (code,label,description,elementtype,rang) values ('FICHINTER_MODIFY','Intervention modify','Executed when a intervention is modify','ficheinter',30);
+insert into llx_c_action_trigger (code,label,description,elementtype,rang) values ('FICHINTER_SIGN','Intervention is signed','Executed when a intervention is set as signed','ficheinter',31);
```

Comment:
To avoid inflation of triggers code, trigger codes must now be limited to CRUD event (Create, update, delete) or Primary status change (but only for the main status). As signature of an intervention is a secondary status, this should already be catched with the FICHINTER_MODIFY (enhanced with $fichinter->context properties) if required.